special-interest-groups/observability-tag/charter.md

```diff
@@ -0,0 +1,48 @@
+# OpenSearch Observability Technical Advisory Group (TAG) Governance Charter
+
+### Purpose
+
+The OpenSearch Project proposes to create a Observability Interest Group that focuses on observability using OpenSearch. This includes log analytics, traces, metrics using OpenSearch and related projects such as Prometheus, OpenTelemetry. This group will also establish guidelines for members, set architecture guidelines, and come up with recommendations which will then be reviewed by TSC for decision making. The group will provide feedback to the TSC before the final decision is made through voting of TSC members.
```

Comment:
I think we should be using TAG vs SIG due to the purpose of this group. I also believe that we should discuss that this is about observing other workloads using the tooling provided by OpenSearch. Although the signals we look at today are the three listed there are likely more signals in the future. Calling out other open source projects might be premature at this juncture. You mention those below so no need here.

Reply:
I agree with using TAG vs SIG. I also agree likely more signals in future. In fact, there is a longer list including profiles later on. However, I like the concrete examples provided here instead of a very generic description. I think, this makes the intent more understandable. The related projects mentioned are already supported by OpenSearch observability as of now. Following this argument Jaeger should probably be added as well.

Reply:
> signal types & related projects

I like explicit call outs. There is nothing stopping us from storing profiles today but the visualization is not there yet. Therefore, I can rephrase as `logs, metrics, traces, and future signals using OpenSearch`.

I will add Jaeger to the list